docs/ugcstor-csi.md

```diff
@@ -5,50 +5,60 @@ sidebar_label: cStor
 ---
 ------
 <br>
-
+ 
 <img src="/docs/assets/svg/3-config-sequence.svg" alt="OpenEBS configuration flow" style="width:100%">
-
+ 
 <br>
 <br>
-
-This user guide section provides the operations needed to be performed by the User and the Admin for configuring cStor related tasks. 
-
-   :::note 
-   The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
-   :::
-
+ 
+This user guide section provides the operations needed to be performed by the User and the Admin for configuring cStor related tasks.
+ 
+  :::note
+  The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
+  :::
+ 
```

Comment:
Remove the user and admin sections. Everything can just flow in as a list of operations. 

Move the snapshot and clone- after the `Scaling up cStor pools`